src/map/tileset.cpp

```diff
@@ -335,6 +344,10 @@ terrain_typeIdx CTileset::getOrAddSolidTileIndexByName(const std::string &name)
 
 const std::string &CTileset::getTerrainName(terrain_typeIdx solidTerrainIndex) const
 {
+	static std::string emptyValue = "";
+	if (solidTerrainIndex >= solidTerrainTypes.size()) {
+		return emptyValue;
```

Comment:
Should it be an error instead?

Reply:
You're right.
fixed: 
https://github.com/Wargus/stratagus/pull/732/commits/4c309350611ae3b65038dc31917f90e9776f1ae4